src/dataflow-types/src/logging.rs

```diff
@@ -269,13 +240,9 @@ impl LogVariant {
             LogVariant::Materialized(MaterializedLog::DataflowCurrent) => vec![],
             LogVariant::Materialized(MaterializedLog::DataflowDependency) => vec![],
             LogVariant::Materialized(MaterializedLog::FrontierCurrent) => vec![],
-            LogVariant::Materialized(MaterializedLog::KafkaBrokerRtt) => vec![(
-                LogVariant::Materialized(MaterializedLog::SourceInfo),
-                vec![(1, 1), (2, 2), (3, 3)],
-            )],
-            LogVariant::Materialized(MaterializedLog::KafkaConsumerInfo) => vec![(
+            LogVariant::Materialized(MaterializedLog::KafkaSourceStatistics) => vec![(
                 LogVariant::Materialized(MaterializedLog::SourceInfo),
-                vec![(1, 1), (2, 2), (3, 3)],
+                vec![(1, 0)],
```

Comment:
I'm not sure that I'm reading the code correctly, but should this be `(0, 1)`?

Reply:
oof, yes, thank you!